packages/core/test/specs/data_sources/jsonplaceholder.ts

```diff
@@ -0,0 +1,103 @@
+import DataSourceManager from '../../../src/data_sources';
+import ComponentWrapper from '../../../src/dom_components/model/ComponentWrapper';
+import { DataVariableType } from '../../../src/data_sources/model/DataVariable';
+import { DataSourceProps } from '../../../src/data_sources/types';
+import { setupTestEditor } from '../../common';
+import EditorModel from '../../../src/editor/model/Editor';
+import htmlFormat from 'pretty';
+
+async function getComments() {
+  const url = 'https://jsonplaceholder.typicode.com/posts/1/comments';
```

Comment:
mmm I'd avoid relying on external services in our tests, let's just get their data as example.